buildSrc/src/main/kotlin/com/virtuslab/gitmachete/buildsrc/UiTests.kt

```diff
@@ -47,13 +48,15 @@ fun Project.configureUiTests() {
         environment("IDEPROBE_DISPLAY", "xvfb")
         environment(
           "IDEPROBE_PATHS_SCREENSHOTS",
-          "${System.getProperty("user.home")}/.ideprobe-uitests/artifacts/uiTest$version/screenshots"
+          "${uiTestsDir}artifacts/uiTest$version/screenshots"
```

Comment:
Nit: I'd move `/` from the `val`
```suggestion
          "${uiTestsDir}/artifacts/uiTest$version/screenshots"
```